telemetry/telemetry_record.py

```diff
@@ -1,23 +1,30 @@
-"""Record WebPageReplay
+"""Record WebPageReplay, modify, and record page load times
 
 Records running a set of target urls on the remote tablet, modifies cacheable
 objects, then re-runs to see page load time differences.
 
-Usage: sudo python record_wpr.py
+Usage: sudo python telemetry_record.py
 
 Notes:
     Assumes - telemetry is in CHROMIUM_SRC/tools/telemetry/telemetry
             - page_sets is in CHROMIUM_SRC/tools/perf/page_sets
             - modify_wpr_delays.py is in CHROMIUM_SRC/third_party/webpagereplay
 
-TODO: Measure page load time before and after modifying
 TODO: Make sure to only get the most recent wpr file when moving / modifying
 TODO: Clean up path dependencies, move them to the top
 """
+from base64 import b64encode
+from os import listdir, path
+from re import findall
 from subprocess import Popen, PIPE, STDOUT
 from sys import path
+import cPickle
+import json
 import os
 import pickle
```

Comment:
random thought: for small amounts of data, I prefer JSON, since it's much easier to debug than binary. But, probably not worth reworking your code, unless you end up doing a lot of debugging later on ;-)